File: app_java/src/main/java/com/dragonforest/demo/app_java/ThirdActivity.java
package com.dragonforest.demo.app_java;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.widget.TextView;

import com.dragonforest.demo.app_java.util.PreUtil;

public class ThirdActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_third);
        initView();
    }

    private void initView() {
        try {
            Thread.sleep(1000);
        } catch (InterruptedException e) {
            e.printStackTrace();
        }
    }

    @Override
    protected void onResume() {
        super.onResume();
        String name ="james";
        String passwd = "heheheh";
        name = PreUtil.INSTANCE.getString(this, PreValues.USER_NAME);
        passwd = PreUtil.INSTANCE.getString(this, PreValues.USER_PASSWD);
        TextView tv_name = findViewById(R.id.tv_name);
        TextView tv_passwd = findViewById(R.id.tv_passwd);
        tv_name.setText(name);
        tv_passwd.setText(passwd);
    }
}

File: app/src/main/java/com/dragonforest/demo/systraceplugin/ThirdActivity.kt
package com.dragonforest.demo.systraceplugin

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.dragonforest.demo.systraceplugin.constants.PreValues
import com.dragonforest.demo.systraceplugin.utils.Monitor
import com.dragonforest.demo.systraceplugin.utils.PreUtil
import kotlinx.android.synthetic.main.activity_third2.*

class ThirdActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_third2)
    }

    override fun onResume() {
        super.onResume()
        var userName = PreUtil.getString1(this,PreValues.USER.USER_NAME)
        var passwd = PreUtil.getString1(this,PreValues.USER.USER_PASSWD)
        tv_name.showMessage(userName)
        tv_name.showMessage(passwd)
        Monitor.record()
    }

}